de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/IInvoiceCandBL.java

```diff
@@ -99,19 +99,34 @@
 	/**
 	 * Creates invoices from the given selection.
 	 * <p>
-	 * <b>IMPORTANT:</b> Candidates with {@link I_C_Invoice_Candidate#isError()} are ignored, even if they are part of the selection!
-	 */
-	IInvoiceGenerateResult generateInvoicesFromSelection(Properties ctx, PInstanceId AD_PInstance_ID, boolean ignoreInvoiceSchedule, String trxName);
-
-	/**
-	 * Creates <code>de.metas.async</code> work packages from for those invoice candidates that are selected via <code>T_Selection</code> with the given <code>AD_PInstance_ID</code>.
-	 *
-	 * @param adPInstance the process instance of the process that does the enqueuing.
-	 * @param onlyApprovedForInvoicing if true, then enqueue only candidates flagged as {@link I_C_Invoice_Candidate#COLUMNNAME_ApprovalForInvoicing}
-	 * @param consolidateApprovedICs <b>ignored, unless <code>onlyApprovedForInvoicing</code> is true </b>. If true then the system will rekey the ICs in question according to the default
-	 *            C_Aggregation, assuming that is is the most basic one with the smallest possible number of items and thus the under under which the most ICs have an equal HeaderAggregationKey.
+	 * <b>IMPORTANT:</b> Candidates with {@link I_C_Invoice_Candidate#isError()} are
```

Comment:
looks like your formatting rule are your of sync this our common practice.
please make sure your have the ones from https://github.com/metasfresh/metasfresh-dev/tree/master/eclipse-config and reformat this code.